### PR TITLE
EE-2340 - Added XML-JSON examples

### PR DESCRIPTION
**Pull request by M-casado**
EE-2340 ticket about adding XML examples and their JSON counterparts.
- [ ] Searched for real XMLs, modified identifiers, and then created their JSON counterparts. 
- [ ] Added XML-JSON examples for policies, sequencing assays and samples.
- [ ] Modified the file object within the common definitions (small optional quality field)
- [ ] Added a README with all the information and instructions on how to validate both formats. 

For the reviewers, I recommend to simply take a look at the README and glance over the examples.

### CODE REVIEW

File: examples/xml-json-same-objects/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+## XML and JSON documents for the same objects
+___
+In this folder you will find **examples of XML and JSON documents that correspond to the same objects**. Each format is validated against their respective schemas: [XSDs](https://github.com/enasequence/schema/tree/master/src/main/resources/uk/ac/ebi/ena/sra/schema) in the case of XMLs and [JSON Schemas](https://github.com/EbiEga/ega-metadata-schema/tree/main/schemas) in the case of JSONs. The aim of this folder is exclusively for you to see how XMLs and JSONs can be used to describe similar sets of data. For example, you will find files [``sample.json``](sample/sample.json) and [``sample.xml``](sample/sample.xml) that both contain the exact same information, but differently formatted. 
+
+These documents are not representations of _good practice_ metadata standards. The metadata they contain is a modified version of XML files once submitted to EGA.
```
Comment (malloryfreeberg) [inline examples/xml-json-same-objects/README.md:5]: 👍

File: examples/xml-json-same-objects/sample/sample.json [hunk 2]
```diff
@@ -0,0 +1,45 @@
+{
```
Comment (malloryfreeberg) [inline examples/xml-json-same-objects/sample/sample.json:1]: The sample.json document especially is so much easier to read than the XML version!

File: examples/xml-json-same-objects/assay/sequencing_assay.json [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+            "sequence_quality_details": {
+                "quality_scoring_system":  "phred"
+            }
+        }
+    ],
+    "assay_relationships": [
+        {
+            "r_type": "referenced_by",
+            "r_source": {
+                "external_accession": "ena.embl:EGAX00002094401"
```
[the remark this answers is not in the document]
Comment (M-casado) [reply examples/xml-json-same-objects/assay/sequencing_assay.json:35]: Yes, you are completely right. I missed that one, but I'll change it in my next commit 👍

File: examples/xml-json-same-objects/sample/sample.json (hunk 2 above)
Comment (M-casado) [reply examples/xml-json-same-objects/sample/sample.json:1]: Glad to hear so 😃

**Review by silviabah — APPROVED** [review]
The documentation is very clear and I was able to validate the examples (XML and json)

**Pull request merged**